shoes-swt/lib/shoes/swt/input_box.rb

```diff
@@ -35,6 +39,10 @@ def enabled(value)
         @real.enabled = value
       end
 
+      def readonly=(value)
+        @readonly = value
+      end
```

Comment:
Good catch. Originally had it as just readonly (no =) like the enabled, so missed that a writer would be better. 👍